src/UI/App.jsx

```diff
@@ -70,16 +74,8 @@ export default class App extends React.Component {
 			this.state.apiCustomHeaders,
 		);
 		this.PollingService = new PollingService(this.Api);
-		this.Api.subscribeDevice(
-			undefined,
-			undefined,
-			undefined,
-			this.state.userAdditionalInformation,
-			DeviceTypes.Web,
-			this.state.deviceVersion,
-		);
-		this.messageIDs = new Set();
-		this.visibilityChange = "visibilitychange";
+
+		Logger.debug("App started, registering device");
```

Comment:
Dat is toch niet waar, je registered het device pas als de chat opent, niet als de app start.

Reply:
Klopt dit was nog overgebleven van toen we subsriben in de constructor. Heb ik nu aangepast